tests/backup/backup_locked_bucket_test.go

```diff
@@ -412,3 +412,236 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+						dash.VerifyFatal(len(appVolumes) > 0, true, "Verifying if app volumes exist")
+						var requestedVols []*volume.Volume
+						log.InfoD(fmt.Sprintf("Increase volume size %s on app %s's volumes",
+							Inst().V.String(), ctx.App.Key))
+						requestedVols, err = Inst().S.ResizeVolume(ctx, Inst().ConfigMap)
+						dash.VerifyFatal(err, nil, "Verifying volume resize")
+						log.InfoD(fmt.Sprintf("validate successful volume size increase on app %s's volumes: %v",
+							ctx.App.Key, appVolumes))
+						for _, volume := range requestedVols {
+							// Need to pass token before validating volume
+							params := make(map[string]string)
+							if Inst().ConfigMap != "" {
+								params["auth-token"], err = Inst().S.GetTokenFromConfigMap(Inst().ConfigMap)
+								dash.VerifyFatal(err, nil, "Fetching token from configMap")
+							}
+							err := Inst().V.ValidateUpdateVolume(volume, params)
+							dash.VerifyFatal(err, nil, "Validate volume update status")
+						}
+					}
+				})
+				Step("Checking size of volume after resize", func() {
+					log.InfoD("Checking size of volume after resize")
+					pods, err := core.Instance().GetPods(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the pod list"))
+					srcClusterConfigPath, err := GetSourceClusterConfigPath()
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
+					for _, pod := range pods.Items {
+						afterSize, err := getSizeOfMountPoint(pod.GetName(), namespace, srcClusterConfigPath)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the mount size %v", afterSize))
+						podListAfterSizeMap[pod.Name] = afterSize
+					}
+					for _, pod := range pods.Items {
+						dash.VerifyFatal(podListAfterSizeMap[pod.Name] > podListBeforeSizeMap[pod.Name], true, fmt.Sprintf("Verifying volume size for pod %s", pod.Name))
+					}
+				})
+				Step("Create schedule backup after initializing volume resize", func() {
+					log.InfoD("Create schedule backup after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					log.FailOnError(err, "Unable to px-central-admin ctx")
+					preRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[i])
+					postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[i])
+					scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+					labelSelectors := make(map[string]string)
+					err = CreateScheduleBackup(scheduleName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+						labelSelectors, orgID, preRuleNameList[i], preRuleUid, postRuleNameList[i], postRuleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
```

Comment:
can we eliminate this duplication of fetching firstScheduleBackupName, as we are calling it while verifiying backup.